cdap-examples/Sports/README.md

```diff
@@ -0,0 +1,30 @@
+# Sports example
+
+Example application that illustrates PartitionedFileSet datasets using sports results.
+
+Features introduced: Partitioned file sets, MapReduce with runtime arguments, Ad-hoh queries over file sets.
+
+- Uses a partitioned file set to store game results. It is partitioned by league and season, and ech partition
+  is a file containing the results of one league for ine season, for example the 2014 season of the NFL.
```

Comment:
though the example you use is from the NBA (National Basketball Association); should this be consistent?